src/hooks/series/useSeriesData.ts

```diff
@@ -1,33 +1,33 @@
 import usePlaylist from '#src/hooks/usePlaylist';
-import useSeries from '#src/hooks/useSeries';
-import type { Playlist } from '#types/playlist';
+import useSeries from '#src/hooks/series/useSeries';
+import type { Playlist, PlaylistItem } from '#types/playlist';
 import type { Series } from '#types/series';
 
 const DEFAULT_DATA = { title: '', playlist: [] };
 
 type Data = {
-  seriesPlaylist: Playlist;
-  series: Series | undefined;
+  playlist: Playlist;
+  newSeries: { series: Series | undefined; media: PlaylistItem | undefined };
 };
 
 export const useSeriesData = (
-  entityId?: string,
+  legacySeriesPlaylistId: string | undefined,
+  seriesId: string | undefined,
 ): {
   data: Data;
```

Comment:
Avoid unnecessary nesting. Also, I think we should always expect to have a media item, right?
```suggestion
  data: {
    series: Series | undefined,
    media: PlaylistItem
  } | undefined;
```

Reply:
We only need it in case it is a new flow, where we get series information from series and media endpoint. Media endpoint is needed to get some metadata.

Reply:
Isn't the media item also used in the old flow? It's just a dummy placeholder with a seriesId custom param.